nemo/core/classes/modelPT.py

```diff
@@ -38,6 +40,7 @@
 from omegaconf import DictConfig, OmegaConf, open_dict
 from pytorch_lightning import LightningModule, Trainer
 from pytorch_lightning.utilities import model_summary, rank_zero_only
+from torch.profiler import ExecutionTraceObserver, _ExperimentalConfig
```

Comment:
Private import should be removed or at least moved directly to where it will be used. If PT changes API of private variable, it will break all Nemo models even if they don't use chakra or profiling

Reply:
thanks, removed to code block in **_setup_chakra_profiling** func in the latest commit